.ado/azure-pipelines.yml

```diff
@@ -110,4 +110,39 @@ jobs:
 
       - template: templates/apple-yarn-build.yml
         parameters:
-          relative_directory: $(relative_directory)
\ No newline at end of file
+          relative_directory: $(relative_directory)
+
+  - job: AndroidPR
+    displayName: Android PR
+    pool:
+      vmImage: 'macos-10.15'
+    variables:
+      relative_directory: 'android'  
+    timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+    cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+    
+    steps:
+      - checkout: self
+        persistCredentials: true
+
+     # setup repo basics
+      - template: templates/setup-repo-min-build.yml
+
+    # package android bundle
+      - script: |
+          yarn bundle
+        workingDirectory: apps/$(relative_directory)
+        displayName: 'yarn bundle $(relative_directory)'
+
+    # sets-up specifics for android dependency like NDK & emulator
+      - template: templates/android-dep-setup.yml
+    
+      # build android apk and run available Intrumented test
+      - task: Gradle@2
+        inputs:
+          gradleWrapperFile: 'apps/$(relative_directory)/src/gradlew'
+          workingDirectory: 'apps/$(relative_directory)/src'
+          tasks: 'assembleDebug cAT'
+          publishJUnitResults: false
+          javaHomeOption: 'JDKVersion'
+          sonarQubeRunAnalysis: false
```

Comment:
Could we add a step that also runs `yarn android`, similar to what we do in `templates/apple-yarn-build.yml` ? That is what we list on the repo's README as the way to launch the android app and I assume is the way newcomers would try and run the android test app. Running that command (which just invokes `react-native run-android`) should also invoke gradle in the process

Reply:
assemble command loads the bundle created by bundle command and also is more reliable and faster than  as we don't need for NPM server to load bundle (which takes time before starting any UI tests.)  Your Thoughts ?

Reply:
I see, that makes sense. I guess we used the run-ios/macos as an initial test case before we wrote any other UI tests. 
If the assemble command is enough to guarantee that `run-android` also works (since it's in our readme) that would be great =).